N24DCCN058/Bai tap buoi 8/Bai4.cpp

```diff
@@ -0,0 +1,18 @@
+#include <bits/stdc++.h>
+
+using namespace std ;
+
+string To_upper(string str){
+    string u ;
+    for (int i = 0 ; i < str.size() ; i++){
+        u += toupper(str[i]);
```

Comment:
chuyển thành chữ in thường, không phải chữ in hoa. đọc lại đề rồi sửa nha

Reply:
dạ